<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>output</class>
 <widget class="QWidget" name="output">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>495</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>118</red>
        <green>200</green>
        <blue>173</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>118</red>
        <green>200</green>
        <blue>173</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>118</red>
        <green>200</green>
        <blue>173</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>118</red>
        <green>200</green>
        <blue>173</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Result</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="output_result"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>File Hash</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="output_hash"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Magic Number</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="output_magic"/>
   </item>
   <item>
    <widget class="QTextEdit" name="output_text"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Disassembled Code</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="output_code"/>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Imported Functions</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="output_functions"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
